retis/src/collect/collect.rs

```diff
@@ -543,8 +548,9 @@ impl Collectors {
 
         let (mut iccount, mut eccount) = (0, 0);
         let mut probe_stack = ProbeStack::new(
-            collect.stack,
-            self.probes.runtime_mut()?.attached_probes(),
+            self.probes
+                .runtime_mut()?
+                .attached_probes_with_opt(ProbeOption::ProbeStack),
```

Comment:
Sorry, missed that. This is not right as what we're looking for is the full list of probes being attached, otherwise if a probe w/o `ProbeStack` is reported in a stack trace, attaching a probe at runtime to it will fail. This chunk can be dropped.

Reply:
This had to be dropped. It is already in my local version, not sure why was not part of this version.